docs/user-guide/release-notes/2.11.x.rst

```diff
@@ -0,0 +1,13 @@
+=======================
+Pulp 2.11 Release Notes
+=======================
+
+Pulp 2.11.0
+===========
+
+New Features
+------------
+
+* Re-sync of the repository will be a no operation by default if since the last sync, no units
```

Comment:
it did not change from user perspective, but i think it still worth to mention that there is behaviour change, that if the content was removed from last sync or config was updated + other conditions that were moved to platform, then a force-sync will be triggered